src/integrator_whfast.c

```diff
@@ -789,6 +788,7 @@ void reb_integrator_whfast_part2(struct reb_simulation* const r){
             struct reb_particle* const particles_var1 = particles + vc.index;
             const int index = vc.index;
             // Centre of mass
+            reb_transformations_inertial_to_jacobi_posvel(ri_whfast->p_jh+index, particles_var1, particles, N_real);
```

Comment:
Is this doing extra work when `keep_unsynchronized=0`?

Reply:
good point!